src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/StartBusinessProcessTaskHandler.java

```diff
@@ -64,6 +64,13 @@ private CaseData startBusinessProcess(ExternalTask externalTask) {
                 }
                 return data;
             default:
+                System.out.println("----------------CAMUNDAERROR -START------------------");
```

Comment:
Why are we using sysouts instead of logger?

Reply:
Used log.error now